### PR TITLE
Task management 02

### LINKED ISSUE #161: reset a task as incomplete

### LINKED ISSUE #162: mark a task as completed

### LINKED ISSUE #163: refine the UI of task list

### LINKED ISSUE #164: read tasks from a text file and parse them to google tasks

### PR DESCRIPTION
**Pull request by EdwardKSG**
refine the UI of task list   close #163 
read tasks stored in a text file and push to the task list in google account   close #164 
add a command to complete a task   close #162 
add a command to reset a task as incomplete   close #161

### CODE REVIEW

File: src/main/java/seedu/progresschecker/logic/parser/CompleteTaskCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package seedu.progresschecker.logic.parser;
+
+import static seedu.progresschecker.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.progresschecker.commons.exceptions.IllegalValueException;
+import seedu.progresschecker.logic.commands.CompleteTaskCommand;
+import seedu.progresschecker.logic.parser.exceptions.ParseException;
+
```
Comment (kohyeeru) [inline src/main/java/seedu/progresschecker/logic/parser/CompleteTaskCommandParser.java:8]: Did you forget to claim this file? :>

File: src/main/java/seedu/progresschecker/logic/parser/ResetTaskCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package seedu.progresschecker.logic.parser;
+
+import static seedu.progresschecker.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.progresschecker.commons.exceptions.IllegalValueException;
+import seedu.progresschecker.logic.commands.ResetTaskCommand;
+import seedu.progresschecker.logic.parser.exceptions.ParseException;
+
```
Comment (kohyeeru) [inline src/main/java/seedu/progresschecker/logic/parser/ResetTaskCommandParser.java:8]: Forgot to claim this too?

**Review by kohyeeru — CHANGES_REQUESTED** [review]
Check if the portions I pointed out are really places that you missed out collating on!

File: src/main/java/seedu/progresschecker/logic/parser/CompleteTaskCommandParser.java (hunk 1 above)
Comment (EdwardKSG) [reply src/main/java/seedu/progresschecker/logic/parser/CompleteTaskCommandParser.java:8]: Thanks for reminding!

**Review by kohyeeru — APPROVED** [review]
Looks good now! 👍

**Review by AdityaA1998 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**